src/twisted/internet/base.py

```diff
@@ -757,26 +839,30 @@ def startRunning(self):
             threadable.registerAsIOThread()
         self.fireSystemEvent("startup")
 
-    def _reallyStartRunning(self):
+    def _reallyStartRunning(self) -> None:
         """
         Method called to transition to the running state.  This should happen
         in the I{during startup} event trigger phase.
         """
         self.running = True
 
-    def run(self):
+    def run(self) -> None:
         # IReactorCore.run
         raise NotImplementedError()
 
     # IReactorTime
 
     seconds = staticmethod(runtimeSeconds)
 
-    def callLater(self, delay, callable: Callable[..., Any], *args, **kw):
-        """See twisted.internet.interfaces.IReactorTime.callLater."""
+    def callLater(
+        self, delay: float, callable: Callable[..., Any], *args: object, **kw: object
+    ) -> DelayedCall:
+        """
+        See twisted.internet.interfaces.IReactorTime.callLater.
+        """
         assert builtins.callable(callable), "{} is not callable".format(callable)
         assert delay >= 0, "{} is not greater than or equal to 0 seconds".format(delay)
-        tple = DelayedCall(
+        delayedCall = DelayedCall(
```

Comment:
What TF was `tple` supposed to mean? I assume that once upon a time, `DelayedCall` has a tuple.